mtp_cashbook/assets-src/stylesheets/elements/_subheader.scss

```diff
@@ -0,0 +1,9 @@
+.global-subheader {
+  padding-top: 12px;
+  padding-bottom: 12px;
+  background: $grey-3-4;
```

Comment:
Can we use a colour from the existing [govuk colour palette](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_colours.scss)?